src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+## 1.9.0-beta.2
+
+Released 2024-Jul-30
+
+* Add `OpenTelemetry.Instrumentation.StackExchangeRedis` Meter
+([#1982](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1982))
+* New Metrics: `redis.client.request.duration`,
+`redis.client.request.waiting_response`, `redis.client.request.time_in_queue`
```

Comment:
Please add entry under unreleased section. The exact version and the release date will be handled later (separate PR).

```suggestion
* Add `OpenTelemetry.Instrumentation.StackExchangeRedis` meter. Supported metrics:
  * redis.client.request.duration`,
  * `redis.client.request.waiting_response`,
  * `redis.client.request.time_in_queue`.
  ([#1982](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1982))
```

Reply:
Done, can you check, please? Thanks

Reply:
Formatting also can be adjusted. See my previous comment.

Reply:
My bad, I missed the formatting. Done.